src/components/application_manager/rpc_plugins/rc_rpc_plugin/test/commands/button_press_request_test.cc

```diff
@@ -153,7 +154,8 @@ class ButtonPressRequestTest
                                         mock_rpc_service_,
                                         mock_hmi_capabilities_,
                                         mock_policy_handler_,
-                                        mock_allocation_manager_);
+                                        mock_allocation_manager_,
+                                        mock_interior_data_cache_);
```

Comment:
@LuxoftAKutsan Why do you use mock_interior_data_cache_ only here? Should't be any EXPECTATIONS for this mock object added?